src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -77,6 +77,9 @@ const ArticlePage = ({ pageData }) => {
     palette: { GREY_2, WHITE },
   } = useTheme();
 
+  const isTransliterated =
```

Comment:
We'll need this logic in the MediaArticlePage component as well as it can also show Topic tags.